Firestore/core/src/firebase/firestore/util/status.h

```diff
@@ -161,6 +198,10 @@ inline Status::State::State(const State& s)
                                                    : s.platform_error->Copy()) {
 }
 
+inline Status::State::State(Error code, std::string msg)
+    : code(std::move(code)), msg(std::move(msg)) {
```

Comment:
There's no need to move `code`, moving built-ins is the same as copying them.

Reply:
Done.